src/rules/__tests__/require-tothrow-message.test.js

```diff
@@ -1,112 +0,0 @@
-import { RuleTester } from 'eslint';
```

Comment:
Why  isn't this a rename?

Reply:
You'll have to ask git that 🤷‍♂

Reply:
> If indentation changed, please revert it.

Sure, but I honestly have no idea what it is:
```
import { RuleTester } from 'eslint';
// vs
import { TSESLint } from '@typescript-eslint/experimental-utils';
```

Maybe line ends? I'll check it out